docs/use/Interact-with-dApps/index.md

```diff
@@ -3,18 +3,44 @@ sidebar_position: 4
 
-## Accessing Evmos Dashboard
+A typical flow to interact with a dApp can look like this:
 
-Once you have your wallet set up with Keplr, head to the [Evmos Dashboard](https://app.evmos.org) to engage with our applications. If you do not have any tokens, you can acquire tokens by reading [these](./../transfer#onramp-evmos) strategies.
+  1. Connect your wallet to the Evmos network.
+  2. Visit a dApp and submit a transaction (e.g. to stake your tokens).
+  3. On your wallet, sign and broadcast the transaction by paying the network fee with gas.
+  4. Wait until the transaction is confirmed and successful.
+  5. View the changes from your transaction in the dApp. You can view more details about your transaction using a block explorer. Transactions are batched and processed together with other transactions in a block and block explorers allow anyone to view all transactions in the network.
+
+## Evmos dApps
+
+Alongside the applications built by the developer community, the Evmos Core Development Team also builds its own [Evmos dApps](https://app.evmos.org). An easy way to get started is by staking and voting for governance proposals.
+
+### Staking
+
+Staking plays an essential role in keeping the Evmos network secure and decentralized. The Evmos network is run by a technology that incentivizes a set of community members to keep it online and decentralized, called Proof-of-Stake. These community members, called validators, are responsible for validating transactions in a block and are usually very technical teams as they have to monitor and upgrade the software that runs the Evmos network. Validators are incentivized to stay online as they earn a commission on your stake.
+
+You can help keep the network secure and put your EVMOS tokens at "stake", by delegating them to a validator. This will earn you a daily interest in form of new EVMOS tokens (aka. staking rewards). As long as your tokens are staked, you cannot transfer or use them in other dApps. At any given time, you can undelegate your tokens. This will allow you to start using your tokens again after an unbonding period that is defined in the protocol (e.g. two weeks).
+
+You can choose validators to stake your tokens with from a list on the [Evmos Staking dApp](https://app.evmos.org/staking). We recommend choosing a reputable validator, that can promise to stay online and doesn't have a large voting power. Validators that are offline for an extended period or act maliciously can get slashed, resulting in a loss of your stake. Also, delegating to validators with less voting power keeps the network more decentralized. You can switch between validators without undelegating your tokens by redelegating them.
```

Comment:
I would emphasize delegating to a validator that is not in the top 20 validators.